utils/hare-shutdown

```diff
@@ -109,23 +108,30 @@ def node_shutdown():
 
 
 def main() -> None:
-    try:
-        opts, args = getopt.getopt(sys.argv[1:], "v",
-                                   ["verbose", "node", "skip-consul-stop"])
-    except getopt.GetoptError as err:
-        sys.exit(err)
-
-    logging.basicConfig(level=logging.INFO, format='%(message)s')
-
-    skip_consul_stop = False
-    for (opt, value) in opts:
-        if opt in ['-v', '--verbose']:
-            logging.basicConfig(level=logging.DEBUG, format='%(message)s')
-        if opt in ['--skip-consul-stop']:
-            skip_consul_stop = True
-        if opt in ['--node']:
-            node_shutdown()
-            return
+
+    parser = argparse.ArgumentParser()
+    parser.add_argument("-v", "--verbose",
+                        help="Increase output verbosity",
+                        action="store_true")
+    parser.add_argument("--skip-consul-stop",
+                        help="Skip consul service stop",
+                        action="store_true")
+    parser.add_argument("--node",
+                        help="Node shutdown",
+                        action="store_true")
+
+    args = parser.parse_args()
+    if args.verbose:
+        logging.basicConfig(level=logging.DEBUG, format='%(message)s')
```

Comment:
Do we want it the other way around, i.e. Info in case of verbose and Debug otherwise?

Reply:
Verbose is usually understood as 'too wordy' or  'reporting (too) much'. 

In case DEBUG level is set, log messages of both INFO and DEBUG will be seen. If INFO level is set, only INFO messages will be seen. So per my understanding, the code makes sense.

Reply:
@mssawant Agree with Konstantin. Verbose should produce more detailed logs.

Reply:
>In case DEBUG level is set, log messages of both INFO and DEBUG will be seen. If INFO level is set, only INFO messages will be seen

@knekrasov, @SwapnilGaonkar7, is this a universal assumption or just with respect to Python? I think in any case we should explicitly state this assumption in form of a comment here itself.

Reply:
@mssawant  Done